doc/DeveloperGuide.md

```diff
@@ -67,6 +67,23 @@ Priority | As a ... | I want to ... | So that I can...
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+#### Use case: Add person
+
+**MSS**
+
+1. User requests to add a person
+2. AddressBook adds the person
+3. User requests to list persons
```

Comment:
Should this be a step? Is listing part of the behavior of adding a person? Remember to not presume what the user would do, but how the user would do it.

(List would be applicable in the delete function because delete requires an index, which requires the user to first use the list command. Add requires no such index.)

Reply:
I see. I assumed that user would like to see the updated name list.